.github/workflows/tests.yaml

```diff
@@ -6,7 +6,7 @@ on:
 
 jobs:
   linting:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
![image](https://user-images.githubusercontent.com/69512295/203541669-3392a57b-4ee1-4ca1-ae93-409564ad09e2.png)

seems like ubuntu latest does not support python 3.8 anymore